immuni_exposure_ingestion/helpers/api.py

```diff
@@ -40,3 +45,47 @@ async def validate_otp_token(otp_sha: str, delete: bool = False) -> OtpData:
     if data is None:
         raise UnauthorizedOtpException()
     return OtpDataSchema().loads(data)
+
+
+def track_upload(f: Callable[..., Coroutine[Any, Any, HTTPResponse]]) -> Callable:
+    """
+    Utility function to track the metrics relative to the upload request.
+    :param f: The upload function to decorate.
+    :return: The decorated function
```

Comment:
_minor_: consistency with the project's documentation:
(i) The definition starts with a low-cap char (i.e., the upload function)
(ii) The definition should end with a dot.

Reply:
The same applies for all added code, but I won't repeat the comment. 🙏